test/extended/etcd/etcd_storage_path.go

```diff
@@ -246,39 +244,34 @@ func testEtcd3StoragePath(t g.GinkgoTInterface, kubeConfig *restclient.Config, e
 		for _, crd := range etcddataCRDs {
 			errs = append(errs, deleteCRD(ctx, crd.Name, delOptions))
 		}
-		if err := errors.NewAggregate(errs); err != nil {
-			t.Fatal(err)
-		}
+		err := errors.NewAggregate(errs)
+		o.Expect(err).NotTo(o.HaveOccurred())
 	}()
 
-	crds := getCRDs(t, crdClient)
+	crds := getCRDs(crdClient)
 
 	mapper := restmapper.NewDeferredDiscoveryRESTMapper(discocache.NewMemCacheClient(kubeClient.Discovery()))
 	mapper.Reset()
 
 	client := &allClient{dynamicClient: dynamic.NewForConfigOrDie(kubeConfig)}
 
-	if _, err := kubeClient.CoreV1().Namespaces().Create(context.Background(), &kapiv1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: testNamespace}}, metav1.CreateOptions{}); err != nil {
-		t.Fatalf("error creating test namespace: %#v", err)
-	}
+	_, err := kubeClient.CoreV1().Namespaces().Create(context.Background(), &kapiv1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: testNamespace}}, metav1.CreateOptions{})
+	o.Expect(err).NotTo(o.HaveOccurred())
+
 	defer func() {
-		if err := kubeClient.CoreV1().Namespaces().Delete(context.Background(), testNamespace, metav1.DeleteOptions{}); err != nil {
-			t.Fatalf("error deleting test namespace: %#v", err)
-		}
+		kubeClient.CoreV1().Namespaces().Delete(context.Background(), testNamespace, metav1.DeleteOptions{})
```

Comment:
`err := `?

Reply:
> `err := `?

don't need it.  We don't do anything with the information.

Reply:
I think @sttts might have been trying to point out that without assigning the return value there is zero point in the subsequent error check, which would be performed on the `err` defined on line 258 that has already been verified to be nil.

If your intent is to ignore any given error, I encourage you to make that decision explicit both in code (i.e. `_ = ...`) and as a comment (e.g. `// Any error in deletion should not fail the test`) to simplify the life of future maintainers. Better still might be to assign the error and log it at warning level so that there is traceability in the event that something strange happens. 

Also, I think govet would have caught this. I've certainly seen the value in openshift/kubernetes, where we inherited the upstream check. It caught a lot of little gotchas in our carries when I got it working as part of the 1.18 rebase.

@tnozicka thoughts on adding govet support to the `make verify` command provided by build-machinery-go?

Reply:
+1

Reply:
> @tnozicka thoughts on adding govet support to the make verify command provided by build-machinery-go?
should be there
https://github.com/openshift/build-machinery-go/blob/4066d02dbe3da15b38378c826c1f886c76359e52/make/golang.mk#L8

Reply:
@tnozicka Hmm, but verify passed. Are the rules we're using too lax then, and it isn't looking for implicitly ignored return values?

Reply:
I am not sure if ignored return value is a vet error. There is  a lot of these in our code base. Have you tried running `go vet` manually and see if it errors out?
(I also prefer explicitly ignoring the error with `_ = ` so the intention is clear.)

Reply:
Can we please address the review issue and be done with it? This code is fishy. Just fix it.